src/Filter/DateTimeToTimestamp.php

```diff
@@ -32,15 +32,14 @@ class DateTimeToTimestamp extends AbstractFilter
      */
     public function filter($value)
     {
-        $ret = null;
```

Comment:
Why do you return null **or** int? Shouldn't it be more like "there's a timestamp or we have a problem (and throw an exception)"?